docker-compose-test.yml

```diff
@@ -15,7 +15,26 @@ services:
       POSTGRES_DB: tests
     networks:
       - keyshade-test
-
+  web:
```

Comment:
You can revert this, since we are using this docker compose to get the environment of testing up and running